CHANGELOG.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+0.6.1 (to be released)
+----------------------
+
+* Support page object serialization, to make it easier to implement caching
+  mechanisms in web-poet frameworks.
```

Comment:
I think we want to release it only after https://github.com/scrapinghub/web-poet/pull/110 is merged. 

* Serialization is not that useful on its own; 
* we're not switching scrapy-poet cache to it yet (right @wRAR?);
* there are some significant changes in the testing PR related to serialization.